manim/animation/transform.py

```diff
@@ -403,6 +532,18 @@ class ApplyMatrix(ApplyPointwiseFunction):
         The origin point for the transform. Defaults to ``ORIGIN``.
     kwargs
         Further keyword arguments that are passed to :class:`ApplyPointwiseFunction`.
+
+    Examples
+    --------
+
+    .. manim:: ApplyMatrixExample
+        :quality: medium
+
+        class ApplyMatrixExample(Scene):
+            def construct(self):
+                matrix = [[1, 2], [3, 4]]
```

Comment:
I think a transformation matrix like `[[1, 1], [0, 2/3]]` would produce an animation that is nicer to look at. This is a matter of taste though.

Reply:
yeah, i will change the values